### ISSUE TITLE
[Bug]: Route Errors

### ISSUE THREAD

**Issue by JDebono2023** [joaopaulolndev/filament-edit-profile#60]
### What happened?

Durig inital setup, the plug in doesn't appear in the main sidebar menu. When I added the menu item option, it returns this eror: Route [filament.workout-planner.pages.edit-profile-page] not defined.

### How to reproduce the bug

Install the plugin via: composer require joaopaulolndev/filament-edit-profile. 

Add in the plugin to the Panel Provider: use Joaopaulolndev\FilamentEditProfile\FilamentEditProfilePlugin;
 ....
->plugins([
    FilamentEditProfilePlugin::make()
])
.....

Add in the user menu to the Panel Provider:
->userMenuItems([
    'profile' => MenuItem::make()
        ->label(fn() => auth()->user()->name)
        ->url(fn (): string => EditProfilePage::getUrl())
        ->icon('heroicon-m-user-circle')
        //If you are using tenancy need to check with the visible method where ->company() is the relation between the user and //tenancy model as you called
        //->visible(function (): bool {
           // return auth()->user()->company()->exists();
        //}),
])

### Package Version

1.0.33

### PHP Version

8.2

### Laravel Version

11.31

### Which operating systems does with happen with?

Windows

### Notes

Filament 3.2. Publishing the views had no effect.

**Comment by hexters** [comment]
Yes, I also experienced the same thing. Maybe there is a best solution for this issue?

**Comment by joaopaulolndev** [comment]
Fixed on v1.0.43, please update and test
